IMC_DataScience/IMC_OOPTutorial/proposal.md

```diff
@@ -0,0 +1,31 @@
+# TEMPLATE
+
+## :fire: Do not edit this file - copy the template and create your own file.
+
+**[Step-By-Step Technical Blog Guide](https://hq.bitproject.org/how-to-write-a-technical-blog/)**
+
+### :pushpin: Step 1
+**TITLE:**    
+Coding for (Not Quite) Dummies: Object Oriented Design
+
+**TOPIC:**    
+Object Oriented Design
+
+**DESCRIPTION (5-7+ sentences):**    
+This presentation is intended to help prospective students learn higher-level concepts related to Computer Science, and bridge the gap from "Khan Academy" to "Computer Scientist." Specifically, this presentation will focus on Object Oriented Design and teaching students the basic concepts and implementations of it in Java. They will learn about constructors, fields and methods, and inheritance, among other things.
+
+### :pushpin: Step 2
+:family: **TARGET AUDIENCE (3-5+ sentences):**    
+The target audience is students who know the basics of Java but not much beyond that. The course will assume the learners are familiar with syntax and basic functions (there are a variety of resources available for people to learn that) and build upon that knowledge. 
+
+### :pushpin: Step 3
+> Outline your learning/teaching structure: 
+
+**Beginning (2-3+ sentences):**    
```

Comment:
Is there is a particular link/set of links you can point someone to if they do not meet the pre-reqs? Some video or set of Khan academy exercises?